### ISSUE TITLE
`tidy-checks=-*` triggers false-positive

### ISSUE THREAD

**Issue by 2bndy5** [cpp-linter/cpp-linter-action#29]
I added this action to a project I'm helping. The project's owner is still learning C++, so I thought this action would help lament best practices...

I tried to disable clang-tidy by specifying 
```yml
          tidy-checks: '-*'
```
but clang-tidy threw some errors which triggered [an empty comment](https://github.com/Mmessiter/LockDownRadioControl/commit/305ad705dbe1e062156cd55bbb292eb8a7f32f1c#commitcomment-58303484).

[The workflow log](https://github.com/Mmessiter/LockDownRadioControl/runs/3946333706?check_suite_focus=true) reveals that clang-tidy didn't like being told to do nothing. The output from clang-tidy (which in this case was the `--help` output) was interpreted as though the user should be notified. 

>Error: no checks enabled.

I'd like to add a check in the _run.py_ (`run_clang_tidy()`) that allows skipping clang-tidy when all checks are disabled (when `tidy-checks=-*`).

**Comment by 2bndy5** [comment]
I just found that only hidden directories (starting with `.`) at repo root are ignored. Meaning a hidden sub-directory would not be ignored. I'm fixing that while I'm debugging.

Also, I'm revising the `get_line_cnt_from_cols()` to open files as binary instead of encoded text. This should better support CRLF & LF line endings. The current implementation might lead to an infinite loop.
